projects/parsers/example.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jan 25 06:55:48 2019
+
+@author: ed
+"""
+
+from loop_report import LoopReport
+import pandas as pd
+import json
+
+# %% file name and path
+file_path = "/Users/ed/Desktop/"
+file_name = "Loop Report 2019-01-16 08-39-03-06-00"
+
+# %% parse file
+lr = LoopReport()
+loop_dict = lr.parse_by_file(
+        path=file_path,
+        file_name="%s.md" % file_name
+)
+
+# %% save parsed file
+with open(file_path + file_name + '-data.json', 'w') as fp:
+    json.dump(loop_dict, fp, sort_keys=True, indent=4)
+
+# %% put data into a dataframe and save
+loop_df = pd.DataFrame(columns=loop_dict.keys(), index=[0])
+loop_df = loop_df.astype('object')
+for k in loop_dict.keys():
+    loop_df[k][0] = loop_dict[k]
+
+loop_df.to_csv(file_path + file_name + '-data-in-columns.csv', index_label="index")
```

Comment:
do we want the example data and example output to be stored to the same location, or should we leave the example data with the test project and put the output in this directory?

Reply:
typically you would put an example code snippet in the readme. I am not sure where the best place would be to put an example file...i would lean towards the test directory? Typically one would look at that test to see how to use it as well.  Im good with either though